scio-elasticsearch/src/main/scala/com/spotify/scio/elasticsearch/package.scala

```diff
@@ -0,0 +1,38 @@
+package com.spotify.scio
+
+import java.time.Duration
+
+import com.spotify.scio.io.Tap
+import com.spotify.scio.values.SCollection
+
+import scala.concurrent.Future
+
+package object elasticsearch {
```

Comment:
Add doc here like the other package objects. The simple "import all" instruction should be enough.